header-only/examples/cython/demo_impl.cpp

```diff
@@ -72,8 +72,8 @@ ArrayBuffers create_demo_array() {
                     NumpyBuilder<int32_t>>>
     > builder(fields_map);
 
-    auto &one_builder = builder.field<Field::one>();
-    auto &two_builder = builder.field<Field::two>();
+    auto &one_builder = builder.content<Field::one>();
```

Comment:
@ManasviGoyal - thanks for spotting this. Indeed we have changed the API of the builders.